pysap/plugins/mri/opt.py

```diff
@@ -0,0 +1,389 @@
+from modopt.opt.proximity import SparseThreshold
+from modopt.opt.linear import Identity
+from modopt.opt.cost import costObj
+from pysap.numerics.linear import Wavelet2
+from pysap.numerics.fourier import FFT2, NFFT2
+from pysap.numerics.gradient import GradSynthesis2, GradAnalysis2
+from pysap.numerics.opt import opt_fb, opt_cv
+from pysap.plugins.mri.parallel_mri.gradient import Grad2D_pMRI
+
+import numpy as np
+
+
+def rec_ista_2d(data, wavelet_name, samples, mu, nb_scale=4, max_iter=150,
+                tol=1e-4, cartesian_sampling=True, uniform_data_shape=None,
```

Comment:
image_shape would have been a better name for this variable
use_acceleration would habe been a better name for the boolean

Reply:
Fixed